paper/paper.bib

```diff
@@ -157,30 +157,30 @@ @article{schulenburg2017
 }
 
 @misc{vectorclass,
-  author = {Fog, Agner},
-  title = {{VCL:} {C++} vector class library},
-  year = {2020},
-  publisher = {GitHub},
-  journal = {GitHub repository},
-  howpublished = {\url{https://github.com/vectorclass/version2}},
-  commit = {46a58414bc6128b61d77186656860ca7b0b961d8},
+ author = {Fog, Agner},
+ title = {{VCL:} {C++} vector class library},
+ year = {2020},
+ publisher = {GitHub},
+ journal = {GitHub repository},
+ howpublished = {\url {https://github.com/vectorclass/version2}},
+ commit = {46a58414bc6128b61d77186656860ca7b0b961d8},
 }
 
 @misc{qsl2021,
-  author = {Astrakhantsev, Nikita},
-  title = {QSL\_at\_QC},
-  year = {2021},
-  publisher = {GitHub},
-  journal = {GitHub repository},
-  howpublished = {\url{https://github.com/nikita-astronaut/QSL\_at\_QC}},
+ author = {Astrakhantsev, Nikita},
+ title = {{QSL\_at\_QC}},
+ year = {2021},
+ publisher = {GitHub},
+ journal = {GitHub repository},
+ howpublished = {\url {https://github.com/nikita-astronaut/QSL\_at\_QC}},
 }
 
 @misc{amd1999,
-  author = {{AMD Corporation}},
-  title = {{AMD} Discloses New Technologies at {Microprocessor Forum} --- Including x86 64-bit Architecture and {Lightning Data Transport}$^{{\sc TM}}$},
-  howpublished = {Press release},
-  day = {5},
-  month = oct,
-  year = {1999},
-  url = {\url{http://www.amd.com/news/prodpr/99105.html}},
+ author = {{AMD Corporation}},
+ title = {{AMD} Discloses New Technologies at {Microprocessor Forum} --- Including x86 64-bit Architecture and {Lightning Data Transport}$^{{\sc TM}}$},
+ howpublished = {Press release},
+ day = {5},
+ month = oct,
+ year = {1999},
+ url = {\url{http://www.amd.com/news/prodpr/99105.html}},
```

Comment:
I'm not sure this is worth a cite, and if so, perhaps a technical manual would be a better reference than the 1999 press release. But I'm not objecting to this choice.

Reply:
I don't particularly like this citation either so if it's not strictly necessary, let's remove it.